src/main/java/org/mastodon/mamut/tomancak/TomancakPlugins.java

```diff
@@ -63,36 +69,39 @@
 @Plugin( type = MamutPlugin.class )
 public class TomancakPlugins extends AbstractContextual implements MamutPlugin
 {
-	private static final String EXPORT_PHYLOXML = "[tomancak] export phyloxml for selection";
-	private static final String FLIP_DESCENDANTS = "[tomancak] flip descendants";
-	private static final String COPY_TAG = "[tomancak] copy tag";
-	private static final String INTERPOLATE_SPOTS = "[tomancak] interpolate spots";
-	private static final String TWEAK_DATASET_PATH = "[tomancak] tweak dataset path";
-	private static final String LABEL_SELECTED_SPOTS = "[tomancak] label spots";
-	private static final String COMPACT_LINEAGE_VIEW = "[tomancak] lineage tree view";
-	private static final String SORT_TREE = "[tomancak] sort tree";
+	private static final String EXPORT_PHYLOXML = "[exports] export phyloxml for selection";
+	private static final String FLIP_DESCENDANTS = "[trees] flip descendants";
+	private static final String COPY_TAG = "copy tag";
+	private static final String INTERPOLATE_SPOTS = "[trees] interpolate missing spots";
+	private static final String LABEL_SELECTED_SPOTS = "[trees] label selected spots";
+	private static final String COMPACT_LINEAGE_VIEW = "[displays] show compact lineage";
+	private static final String SORT_TREE = "[trees] sort lineage tree";
+	private static final String MERGE_PROJECTS = "merge projects";
+	private static final String TWEAK_DATASET_PATH = "fix project image path";
```

Comment:
I have no strong opinion here, but maybe let's keep the `"[tomancak] ..."` prefix here. It's more consistent, and gives an indication in the settings dialog that this plugin belongs to mastodon-tomancak.

Reply:
okay, changed that back to [tomancak]... I also don't have a strong opinion on that

the original idea was to be able to group/filter related activities with a single keyword.. and so was essentially mirroring the menu layout in the brackets (besides, now that this repo is among the default installed, user is probably not realizing/not cares (maybe?)/does not see the source of this functinality)

but words like lineage, spot, sort... tend to group reasonably well and a user can see everything relevant and adjust/overview shortcuts in the key bindings dialog... some my primary goal is preserved and thus I'm happy :-)

I've took care of it in the commit dde59b7af6c